auth-api/src/auth_api/resources/v1/org.py

```diff
@@ -400,7 +400,12 @@ def get_org_details_by_affiliation(business_identifier):
     try:
         data = OrgService.search_orgs_by_affiliation(business_identifier, pagination_info, environment)
 
-        org_details = [{'name': org.name, 'uuid': org.uuid} for org in data['orgs']]
+        org_details = \
+            [
+                {'name': org.name, 'uuid': org.uuid, 'branchName': org.branch_name} for org
+                in data['orgs']
+                if org.type_code.upper() not in ['SBC_STAFF', 'STAFF']
```

Comment:
I believe there is an org type enum you can use.

Reply:
Does this change affect the pagination since you are potentially only including a subset of orgs after querying?

Reply:
I'd tuck this into the service, I think it's only your code using `search_orgs_by_affiliation`

Reply:
Updated so it takes `excluded_org_types` as param.